src/main/java/com/targus/ui/widgets/Sensor.java

```diff
@@ -8,19 +8,56 @@
 public class Sensor extends Group {
     private static double COMMUNICATION_RADIUS;
     private static double SENSING_RADIUS;
+    private static boolean STATIC_VARIABLES_INITIALIZED = false;
+    private Circle sensor;
+
+    public static void initializeRadii(double communicationRadius, double sensingRadius) {
```

Comment:
Why do not we call this `initializeRadius`?

Reply:
The initialization of the static variables of Sensor.java turned out to be more tricky than it looks. The solve method in InputsController will be refactored and maybe will result in the creation of new classes. Until then, I left that code there to act as a reminder. Unfortunately the implementation does not happen in a linear fashion but rather in a collective fashion such that some implementations depend on the other implementations being done.